### PR TITLE
[SYCL] Fix missed hostData in buffer constructor

### PR DESCRIPTION
**Pull request by KseniyaTikhomirova**
Signed-off-by: Tikhomirova, Kseniya <kseniya.tikhomirova@intel.com>

### CODE REVIEW

**Comment by KseniyaTikhomirova** [comment]
@intel/llvm-reviewers-runtime hello, could you please review? it is a tiny change

File: sycl/include/sycl/buffer.hpp [hunk 1]
```diff
@@ -535,11 +539,16 @@ class buffer : public detail::buffer_plain,
...
+  accessor<
+      T, dimensions, mode, access::target::host_buffer,
+      access::placeholder::false_t,
+      ext::oneapi::
+          accessor_property_list<>> get_access(range<dimensions> accessRange,
+                                               id<dimensions> accessOffset = {},
+                                               const detail::code_location
+                                                   CodeLoc =
+                                                       detail::code_location::
+                                                           current()) {
```
Comment (steffenlarsen) [inline sycl/include/sycl/buffer.hpp:551]: I assume this is just clang-format having changed its mind since last it was run on this?

**Review by steffenlarsen — APPROVED** [review]
LGTM! 🚀

File: sycl/include/sycl/buffer.hpp (hunk 1 above)
Comment (KseniyaTikhomirova) [reply sycl/include/sycl/buffer.hpp:551]: yes, it is

**Pull request merged**